descriptor.schema.json

```diff
@@ -175,21 +175,102 @@
+                        "description": "Permitted choices for input value. May only be used in conjunction with the enum type.",
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "integer": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/integer",
+                        "description": "Specify whether the input should be an integer. May only be used with Number type inputs.",
+                        "type": "boolean"
+                    },
+                    "minimum": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/minimum",
+                        "description": "Specify the minimum value of the input (inclusive). May only be used with Number type inputs.",
+                        "type": "number"
+                    },
+                    "maximum": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/maximum",
+                        "description": "Specify the maximum value of the input (inclusive). May only be used with Number type inputs.",
+                        "type": "number"
+                    },
+                    "exclusive-minimum": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/exclusive-minimum",
+                        "description": "Specify whether the minimum is exclusive or not. May only be used with Number type inputs.",
+                        "type": "boolean"
+                    },
+                    "exclusive-maximum": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/exclusive-maximum",
+                        "description": "Specify whether the maximum is exclusive or not. May only be used with Number type inputs.",
+                        "type": "boolean"
+                    },
+                    "min-list-entries": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/min-list-entries",
+                        "description": "Specify the minimum number of entries in the list. May only be used with List type inputs.",
+                        "type": "number"
+                    },
+                    "max-list-entries": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/max-list-entries",
+                        "description": "Specify the maximum number of entries in the list. May only be used with List type inputs.",
+                        "type": "number"
                     }
-                    
                 },
                 "required": [
                     "name",
                     "id",
                     "type"
-                ]
+                ],
+                "additionalProperties": true,
+                "dependencies": {
```

Comment:
Great, I didn't know that JSON schema allowed for such dependencies. That's neat.